scripts/loket-to-lmb-transform/deleteUnwantedInstances.ts

```diff
@@ -0,0 +1,194 @@
+        ?s ?p ?o.
+      }
+      FILTER NOT EXISTS {
+        ?org org:hasPost ?s.
+        ?org a besluit:Bestuursorgaan.
+      }
+    }`;
+
+  await updateSudo(query, {}, sparqlOptions);
+}
+
+async function deleteUnwantedMandatarissen() {
+  const query = `
+    PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+    PREFIX org: <http://www.w3.org/ns/org#>
+    PREFIX persoon: <http://data.vlaanderen.be/ns/persoon#>
+    PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+
+    DELETE {
+      GRAPH ?g {
+        ?s ?p ?o.
+      }
+    }WHERE {
+      ?s a mandaat:Mandataris.
+      GRAPH ?g {
+        ?s ?p ?o.
+      }
+      FILTER NOT EXISTS {
+        ?org org:hasPost ?post.
+        ?s org:holds ?post.
+        ?org a besluit:Bestuursorgaan.
+      }
+    }`;
+
+  await updateSudo(query, {}, sparqlOptions);
+}
+
+async function deleteUnwantedPersons(){
+  const query = `
+    PREFIX person: <http://www.w3.org/ns/person#>
+    PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+    DELETE {
+      GRAPH ?g {
+        ?s ?p ?o.
+      }
+    }
+    WHERE {
+      ?s a person:Person.
+      GRAPH ?g {
+        ?s ?p ?o.
+      }
+      FILTER NOT EXISTS {
+        {
+          ?m mandaat:isBestuurlijkeAliasVan ?s.
+        }
+        UNION
+        {
+          ?l mandaat:heeftKandidaat ?s.
+        }
+      }
+    }`;
+
+  await updateSudo(query, {}, sparqlOptions);
+}
+
+async function deleteUnwantedBestuurseenheden() {
+  const query = `
+    PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+    DELETE {
+      GRAPH ?g {
+        ?s ?p ?o.
+      }
+    }
+    WHERE {
+      ?s a besluit:Bestuurseenheid.
+      GRAPH ?g {
+        ?s ?p ?o.
+      }
+      FILTER NOT EXISTS {
+        ?org besluit:bestuurt ?s.
+      }
+    } `;
+    await updateSudo(query, {}, sparqlOptions);
+}
+
+async function deleteUnwantedGeboorten(){
+  const query = `
+  PREFIX persoon: <http://data.vlaanderen.be/ns/persoon#>
+  DELETE {
+    GRAPH ?g {
+      ?s ?p ?o.
+    }
+  }
+  WHERE {
+    ?s a persoon:Geboorte.
+    GRAPH ?g {
+      ?s ?p ?o.
+    }
+    FILTER NOT EXISTS {
+      ?pers persoon:heeftGeboorte ?s.
+    }
+  }`;
+  await updateSudo(query, {}, sparqlOptions);
+}
+
+async function deleteUnwantedIdentifiers(){
+  const query = `
+    PREFIX adms: <http://www.w3.org/ns/adms#>
+    DELETE {
+      GRAPH ?g {
+        ?s ?p ?o.
+      }
+    }
+    WHERE {
+      ?s a adms:Identifier.
+      GRAPH ?g {
+        ?s ?p ?o.
+      }
+      FILTER NOT EXISTS {
+        ?pers adms:identifier ?s.
+      }
+    }`;
+  await updateSudo(query, {}, sparqlOptions);
+}
+
+export async function deleteUnwantedInstances(){
+  console.log("Deleting unwanted instances...");
+  await deleteUnwantedBestuursOrgs();
+  await deleteUnwantedMandates();
+  await deleteUnwantedMandatarissen();
+  await deleteUnwantedPersons();
+  await deleteUnwantedBestuurseenheden();
+  await deleteUnwantedGeboorten();
+  await deleteUnwantedIdentifiers();
```

Comment:
Not sure if we delete everything that is unwanted, what about memberships en rechtstreekseVerkiezingen that are linked to resources that are deleted here (I think the specific resouces we delete won't have these relations, but I'm not entirely sure)

Reply:
Memberships are for fractions which only exist for our domain (not for erediensten etc that these queries target). Same for verkiezingen